DTXManiaプロジェクト/コード/全体/CDTXMania.cs

```diff
@@ -2089,7 +2096,10 @@ private void t起動処理()
 				);
 				//Sound管理 = FDK.CSound管理.Instance;
 				//Sound管理.t初期化( soundDeviceType, 0, 0, CDTXMania.ConfigIni.nASIODevice, base.Window.Handle );
-	
+
+				SoundGroupLevelController = new SoundGroupLevelController(CSound.listインスタンス);
```

Comment:
Here we initialize the new SoundGroupLevelController and give it CSound's list of sounds (as maintained by the Sound manager.) Note that SoundGroupLevelController knows nothing about CConfigIni. The ConfigIniToSoundGroupLevelControllerBinder does know how to tie the two together via events and member invocations. Take a look at that class to see how that is achieved.